run/service-auth/receive_test.py

```diff
@@ -25,33 +25,55 @@
 from requests.adapters import HTTPAdapter
 from requests.packages.urllib3.util.retry import Retry
 
-
-@pytest.fixture()
-def services():
-    # Unique suffix to create distinct service names
-    suffix = uuid.uuid4().hex
-    service_name = f"receive-{suffix}"
-    project = os.environ["GOOGLE_CLOUD_PROJECT"]
-
-    # Deploy receive Cloud Run Service
+PROJECT_ID = os.environ["GOOGLE_CLOUD_PROJECT"]
+
+HTTP_STATUS_OK = 200
+HTTP_STATUS_BAD_REQUEST = 400
+HTTP_STATUS_UNAUTHORIZED = 401
+HTTP_STATUS_FORBIDDEN = 403
+HTTP_STATUS_NOT_FOUND = 404
+HTTP_STATUS_INTERNAL_SERVER_ERROR = 500
+HTTP_STATUS_BAD_GATEWAY = 502
+HTTP_STATUS_SERVICE_UNAVAILABLE = 503
+HTTP_STATUS_GATEWAY_TIMEOUT = 504
+
+STATUS_FORCELIST = [
+    HTTP_STATUS_BAD_REQUEST,
+    HTTP_STATUS_UNAUTHORIZED,
+    HTTP_STATUS_FORBIDDEN,
+    HTTP_STATUS_NOT_FOUND,
+    HTTP_STATUS_INTERNAL_SERVER_ERROR,
+    HTTP_STATUS_BAD_GATEWAY,
+    HTTP_STATUS_SERVICE_UNAVAILABLE,
+    HTTP_STATUS_GATEWAY_TIMEOUT,
+],
```

Comment:
HTTP status codes aren't really "magic numbers", but you're also creating a bunch of manual variables for things that are mostly only directly used in your "STATUS_FORCELIST"

Instead of re-creating your own magic variables, you can use the existing lists from either the core standard library [`HTTPStatus`](https://docs.python.org/3/library/http.html#http.HTTPStatus), or from [`requests.codes`](https://requests.readthedocs.io/en/latest/api/#requests.codes), since that'a already imported. Use these to declare your list (as it is called multiple times)